tempodb/encoding/vparquet2/block_traceql.go

```diff
@@ -781,10 +785,13 @@ func checkConditions(conditions []traceql.Condition) error {
 		}
 
 		// Check for conditions that are not supported in vParquet2
-		if cond.Attribute.Intrinsic == traceql.IntrinsicEventName {
+		if cond.Attribute.Intrinsic == traceql.IntrinsicEventName ||
```

Comment:
At some point I would like to swap checkConditions to look for supported things, instead of unsupported things.  Which is less brittle going forward.

Reply:
I agree, it would be safer to check what is supported. I've made a note for myself to refactor this in a separate PR